Quasardb/GetBytes.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace Quasardb
+{
+    /// <summary>
+    /// Helper class
+    /// </summary>
+    public static class Helper
+    {
+        /// <summary>
+        /// Safely Get Bytes
+        /// </summary>
+        public static byte[] GetBytes(int size, IntPtr Pointer)
```

Comment:
Why size+ptr instead of ptr+size as usual?

Reply:
Correct.